.github/workflows/python-test.yml

```
@@ -142,42 +138,50 @@ jobs:
+          - "3.11"
+          - "3.12"
+        exclude:
+          # Reduce costs by testing one version of Python on MacOS
+          - os: "macos-13"
+            python-version: "3.10"
+          - os: "macos-13"
+            python-version: "3.11"
+    runs-on: ${{ matrix.os }}
+    if: "!startsWith(github.event.head_commit.message, 'Version updated')"
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)